packages/react-native/src/handlers/index.ts

```diff
@@ -0,0 +1,15 @@
+import { Platform } from 'react-native';
+import { AndroidUnhandledExceptionHandler } from './android/AndroidUnhandledExceptionHandler';
+import { type ExceptionHandler } from './ExceptionHandler';
+import { UnhandledExceptionHandler } from './UnhandledExceptionHandler';
+
+export function generateUnhandledExceptionHandler(): ExceptionHandler {
+    switch (Platform.OS) {
+        case 'android': {
+            return new AndroidUnhandledExceptionHandler();
+        }
+        default: {
```

Comment:
No iOS love? 😢

Reply:
Every swift/objective-c error will cause an application crash. In Java managed errors might cause it, but crashpad won't capture them. When the managed environment detects an error we can still call javascript env and generate a report. On iOS the same thing is impossible.


There is also a change we extend the offer by adding more platforms later, so I believe its useful to have it here